reducers/views/rhs.js

```diff
@@ -241,4 +277,8 @@ export default combineReducers({
     isSidebarOpen,
     isSidebarExpanded,
     isMenuOpen,
+    shouldShowPreviewOnCreateComment,
```

Comment:
This isn't really the right place to put these since this file is specifically supposed to be for the RHS. We'd like to keep them separated out so that it's easier to find them in the future. Could you maybe move these into a new file like `reducers/views/textbox.js`?